web/src/components/table/use-cases/sessions.tsx

```diff
@@ -58,7 +58,7 @@ export default function SessionsTable({
   const userIdFilter: FilterState = userId
```

Comment:
<img width="523" alt="image" src="https://github.com/langfuse/langfuse/assets/17686849/32852e5e-3553-4734-8ecf-234b73036552">

Do you know why the selection in the dropdown is not selected?

Reply:
Found the issue and fixed it. 👍

Reply:
Fixed both